config/initializers/id_mapper.rb

```diff
@@ -0,0 +1,3 @@
+require 'id_mapper'
+
+IDMapper.logger = Logger.new(STDOUT) if Rails.env.development?
```

Comment:
Is there any reason not to use `Rails.logger` here?

Reply:
No, think I took this from some id-mapper code which is obviously Rails-less. I'll push a fixup